rtwo/drivers/openstack_network.py

```diff
@@ -38,8 +39,17 @@ def new_connection(self, *args, **kwargs):
         Allows us to make another connection (As the user)
         """
         #HACK - Nova is certified-broken-on-v3. 
-        kwargs['version'] = 'v2.0'
-        kwargs['auth_url'] = kwargs['auth_url'].replace('v3','v2.0')
+        #RBB note: The MOC Nova is not broken on v3
+        #kwargs['version'] = 'v2.0'
+        #kwargs['auth_url'] = kwargs['auth_url'].replace('v3','v2.0')
+        #somewhere there is a filter that converted the kwargs
+        #as I don't have time to find it, I am just going to convert things back 
+        if kwargs.has_key(u'token') and not kwargs.has_key(u'ex_force_auth_token'):
+            kwargs[u'ex_force_auth_token']=kwargs[u'token']
```

Comment:
`git grep` is your friend here -- The "filter" that was used is [Copy Connection](https://github.com/cyverse/rtwo/blob/ede32e824245b85e6c70fe35f0c0ad312eafe3b4/rtwo/drivers/openstack_facade.py#L332)

You are in control of the `kwargs` that are passed into these new_connections, so why convert `_ex_force_auth_token --> token --> ex_force_auth_token`?
`_ex_force_auth_url --> auth_url --> ex_force_auth_url`?